gems/manageiq_ansible_tower/manageiq_ansible_tower.gemspec

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'manageiq_ansible_tower/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "manageiq_ansible_tower"
+  spec.version       = ManageiqAnsibleTower::VERSION
+  spec.authors       = ["Brandon Dunne"]
+  spec.email         = ["bdunne@redhat.com"]
+
+  spec.summary       = %q{Ansible Tower Provider for ManageIQ}
+  spec.description   = %q{Ansible Tower Provider for ManageIQ}
```

Comment:
Techniclly this is not the provider, right?  This is just the Ruby interface to Ansible Tower.  If it isn't, I recommened creating two separate gems.  One that is just a REST API wrapper for Ansible Tower, and a *separate* one for the manageiq provider.

Also, for naming, it will save a lot of hassle in the future if the provider plugin is named `manageiq-providers-ansible_tower` @matthewd Can you confirm what I'm saying here?